CRM/Utils/System/Joomla.php

```diff
@@ -841,7 +841,7 @@ public function outputError($content) {
    * @param array $list
    */
   public function appendCoreResources(&$list) {
-    $list[] = 'js/crm.joomla.js';
+    CRM_Core_Resources::singleton()->addScriptFile('civicrm', 'js/crm.joomla.js');
```

Comment:
@monishdeb Why is this different to the rest?  Can we have a comment if there is a good reason

Reply:
It's a race-condition fix. As I understand it, Monish did this to move `crm.joomla.js` to the footer so it will load after the menubar.

Reply:
@monishdeb I agree this needs a comment.